src/DataTable.vue

```diff
@@ -255,6 +253,36 @@
 		},
 
 		computed: {
+			perPageOptions: function() {
```

Comment:
This can be very significantly shortened with arrow-functions, and the first few lines can just be

```
var options = (Array.isArray(perPage) && perPage) || [10, 20, 30, 40, 50];
```